presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/EmbeddedKafka.java

```diff
@@ -91,7 +92,7 @@ public static EmbeddedKafka createEmbeddedKafka(Properties overrideProperties)
                 .build();
 
         KafkaConfig config = new KafkaConfig(properties);
-        this.kafkaServer = new KafkaServer(config, Time.SYSTEM, Option.empty(), List.make(0, null));
+        this.kafkaServer = new KafkaServer(config, Time.SYSTEM, Option.empty(), KafkaMetricsReporter$.MODULE$.startReporters(new VerifiableProperties(new Properties())));
```

Comment:
What is this?

Reply:
the new version needs a KafkaMetricReporter..
changed to use java version..